lib/fix-me.js

```diff
@@ -55,12 +54,12 @@ FixMe.prototype.find = function(file, strings){
 
       lines.forEach(function(line, index, array){
         // grep spits out an extra line that we can ignore
-        if(index < (array.length-1)){
+        if (index < (array.length-1)) {
           // Grep output is colon delimited
           var cols = line.split(":");
 
           // Remove remnants of container paths for external display
-          var fileName = cols[0].split("/code/")[1];
+          var fileName = cols[0].split("/code/").slice(1).join("/code/");
           var lineNum = cols[1];
```

Comment:
@GordonDiggs do you think `replace` + regex for `^/code/` is better here?

I can go ahead and make switch if that's cleaner